src/plugins/ceramic-feed/server/content-types/ceramic-post/schema.js

```diff
@@ -1,60 +1,57 @@
 module.exports = {
-  info: {
-    tableName: 'ceramic-post',
-    singularName: 'ceramic-post',
-    pluralName: 'ceramic-posts',
-    displayName: 'Ceramic feed post',
-    description: 'Live articles piblished to the ceramic network stream',
-    kind: 'collectionType',
-  },
-  options: {
-    draftAndPublish: false,
-  },
-  pluginOptions: {
-    'content-type-builder': {
-      visible: false,
-    },
-  },
-  attributes: {
-    title: {
-      type: 'string',
-      required: true,
-      configurable: false,
-    },
-    content: {
-      type: 'richtext',
-      required: true,
-      configurable: false,
-    },
-    picture: {
-      type: 'media',
-      multiple: false,
-      required: false,
-      configurable: false,
-      allowedTypes: ['images'],
-    },
-    sponsoredLink: {
-      type: 'string',
-      required: false,
-      configurable: false,
-    },
-    sponsoredLogo: {
-      type: 'media',
-      multiple: false,
-      required: false,
-      configurable: false,
-      allowedTypes: ['images'],
-    },
-    link: {
-	  type: 'string',
-      required: true,
-      configurable: false,
-    },
-    cid: {
-      type: 'string',
-      private: true,
-      unique: true,
-      configurable: false,
-    },
-  },
-};
\ No newline at end of file
+	'kind': 'collectionType',
+	'info': {
+	  'tableName': 'ceramic-post',
+	  'singularName': 'ceramic-post',
+	  'pluralName': 'ceramic-posts',
+	  'displayName': 'Ceramic feed post',
+	  'description': 'Live articles piblished to the ceramic network stream',
+	  'kind': 'collectionType'
+	},
+	'options': {
+	  'draftAndPublish': false
+	},
+	'pluginOptions': {
+	  'content-type-builder': {
+		'visible': true
+	  }
+	},
+	'attributes': {
+	  'title': {
+		'type': 'string',
+		'required': true,
+		'configurable': false
+	  },
+	  'content': {
+		'type': 'richtext',
+		'required': true,
+		'configurable': false
+	  },
+	  'picture': {
+		'type': 'media',
+		'multiple': false,
+		'required': false,
+		'configurable': false,
+		'allowedTypes': [
+		  'images'
+		]
+	  },
+	  'link': {
+		'type': 'string',
+		'required': true,
+		'configurable': false
+	  },
+	  'cid': {
+		'type': 'string',
+		'private': true,
+		'unique': true,
+		'configurable': false
+	  },
+	  'sponsor': {
+		'type': 'component',
```

Comment:
should be relation Many-To-One
please read the docs: https://docs.strapi.io/developer-docs/latest/development/backend-customization/models.html#relations